lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -1013,6 +1092,48 @@ def test_error(self):
         response_data = json.loads(response.content)
         self.assertEqual(response_data, expected_response_data)
 
+    @ddt.data(
+        ("abuse_flagged", True),
+        ("abuse_flagged", False),
+    )
+    @ddt.unpack
+    def test_closed_thread(self, field, value):
+        self.register_thread({"closed": True})
+        self.register_comment()
+        self.register_flag_response("comment", "test_comment")
+        request_data = {field: value}
+        response = self.client.patch(  # pylint: disable=no-member
+            self.url,
+            json.dumps(request_data),
+            content_type="application/json"
+        )
+        self.assertEqual(response.status_code, 200)
+        response_data = json.loads(response.content)
+        self.assertEqual(
+            response_data,
+            self.expected_response_data({
+                "abuse_flagged": value,
+                "editable_fields": ["abuse_flagged"],
+            })
+        )
+
+    @ddt.data(
+        ("raw_body", "Edited body"),
+        ("voted", True),
+        ("following", True),
+    )
+    @ddt.unpack
+    def test_closed_thread_error(self, field, value):
+        self.register_thread({"closed": True})
+        self.register_comment()
+        request_data = {field: value}
+        response = self.client.patch(  # pylint: disable=no-member
+            self.url,
+            json.dumps(request_data),
+            content_type="application/json"
+        )
```

Comment:
Nit: You may want to create a common helper method in the base class for patching the URL.  This way, there would be only one place that would be updated once the `content_type` changes.